demos/projects/ESPRESSIF/esp32/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# The following lines of boilerplate have to be in your project's CMakeLists
+# in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.5)
```

Comment:
>cmake_minimum_required [](http://example.com/codeflow?start=0&length=22)

our repo mostly support 3.13, is this something we should keep consistent with main repo?

Reply:
For the esp32 sample, for as long as it lives in iot-middleware-freertos-samples, it should follow what the samples structure defines.
So LSS, yes. I'll update the version.